assets/expo-level/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "expo-level",
+  "version": "0.3.2",
+  "description": "level adapter for react-native-leveldb",
+  "main": "lib/commonjs/index",
+  "module": "lib/module/index",
+  "types": "lib/typescript/index.d.ts",
+  "react-native": "src/index",
+  "source": "src/index",
+  "files": [
+    "src",
+    "lib"
+  ],
+  "scripts": {
+    "typescript": "tsc --noEmit",
+    "prepare": "bob build",
+    "release": "release-it",
+    "test": "jest",
+    "testapp": "cd testapp && npm run start"
+  },
+  "keywords": [
+    "react-native",
+    "ios",
+    "android"
+  ],
+  "repository": "https://github.com/ImSingee/expo-level",
+  "author": "Bryan",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ImSingee/expo-level/issues"
+  },
+  "homepage": "https://github.com/ImSingee/expo-level",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "abstract-level": "^1.0.3",
+    "events": "^3.3.0"
+  },
+  "peerDependencies": {
+    "expo-file-system": "*",
+    "react": "*",
+    "react-native": "*",
+    "text-encoding": "*",
+    "react-native-leveldb": "^3.1.0"
+  },
+  "peerDependenciesMeta": {
+    "text-encoding": {
+      "optional": true
+    }
+  },
+  "devDependencies": {
+    "@commitlint/config-conventional": "^11.0.0",
+    "@release-it/conventional-changelog": "^2.0.0",
+    "@types/react": "17.0.2",
+    "@types/react-native": "0.68.2",
+    "commitlint": "^11.0.0",
+    "react-native-builder-bob": "^0.18.0",
```

Comment:
interesting that `react-native-builder-bob` was used for this as the project is pure JS. we can probably edit in the future to remove all of this boilerplate if we fork